lib/helper/Puppeteer.js

```diff
@@ -1090,7 +1090,11 @@ class Puppeteer extends Helper {
     for (const modifier of modifiers) {
       await this.page.keyboard.down(modifier);
     }
-    await this.page.keyboard.press(key);
+    try {
+      await this.page.keyboard.press(key);
+    } catch (e) {
```

Comment:
Please rewrite this without using try/catch, so you detect if this is a special char and if not, then type the value in

Reply:
I try to detect the special characters now with the keyDefinitions provided by puppeteer. As I am not aware of any edge cases, I did not add additional tests.